### ISSUE TITLE
Try DI via PropertyWrappers

### ISSUE THREAD

**Issue by fchiusolo** [Donald90/RandomJokes#14]

**Comment by dcampogiani** [comment]
Any update?

**Comment by dcampogiani** [comment]
Any update?

**Comment by fchiusolo** [comment]
Yes, I took a look in the weekend but I still didn't manage to do it

**Comment by dcampogiani** [comment]
@Donald90 ok, thanks for the reply.

**Comment by dcampogiani** [comment]
Hello Francesco,

I hope this message finds you well! It's been a while since you opened this issue. The world has been eagerly awaiting updates on this front, as the anticipation grows by the day. It seems like ages have passed since you initially brought up this intriguing idea, and it's understandable that life has been quite the whirlwind since then.

You've been on quite the journey, haven't you? Moving to a new country is no small feat, and it's admirable how you're navigating the challenges of settling into a new place while continuing to grow both personally and professionally. It's fascinating to think about how these experiences might be influencing your perspective on software development, perhaps even inspiring new and innovative ideas for RandomJokes.

Remember that one time when you had that close call with the burger? I can't help but chuckle thinking back to that moment. It was such a hilarious episode—a perfect example of how life can throw unexpected surprises our way! The image of you dashing home just in time is a classic story that's bound to bring a smile to anyone's face. It's these little life adventures that often remind us to find humor in the everyday, much like the spirit of the RandomJokes app.

So, with all these stories and experiences in mind, we're all curious: have there been any new thoughts or developments regarding the use of Dependency Injection via Property Wrappers? The community is really excited to hear about any progress or insights you might have. Whether it's a new approach, a shift in priorities, or simply a pause in the project, we'd love to hear from you and understand where things stand.

Take your time, of course—life comes first, and we all understand that. But if you find a moment amidst your adventures and new beginnings, it would be wonderful to get an update. After all, the whole world (or at least your dedicated followers!) is waiting with bated breath to see what comes next in the RandomJokes saga.

Wishing you all the best in your new endeavors and looking forward to hearing from you soon!

Cheers!
